src/pushsource/_impl/backend/koji_source.py

```diff
@@ -244,6 +265,15 @@ def _push_items_from_rpm_meta(self, rpm, meta):
             )
         ]
 
+    def _module_filtered(self, file_path):
+        ok_names = self._module_filter_filename
+        filename = os.path.basename(file_path)
+        if ok_names != None and filename not in ok_names:
```

Comment:
Not a big deal I guess, but `ok_names is not None ` should be used when comparing to None or probably only ` if ok_names and filename not in ok_names:` can be used as well.

Reply:
I agree on the "is not None", fixed it.

I don't think it would be right to make it just "if ok_names and ...".  The reason is that it loses the distinction between "caller didn't request any filtering" and "caller requested filtering to an empty list".  Asking to filter to an empty list might seem weird since you're always going to get nothing back, but in the long run I think treating an empty list differently to lists of any other length will end up more confusing.